using-mobile-extensions/adobe-analytics/README.md

```diff
@@ -704,9 +822,13 @@ NSDictionary *updatedConfig = @{@"analytics.server":@"sample.analytics.tracking.
                                 @"analytics.rsids":@"rsid1,rsid2",
                                 @"analytics.batchLimit":@(10),
                                 @"analytics.offlineEnabled":@YES};
-[ACPCore updateConfiguration:updatedConfig];
+[AEPMobileCore updateConfiguration:updatedConfig];
 ```
 
+{% endtab %}
+
+{% tab title="iOS (ACP 2.x) %}
```

Comment:
Is this missing a terminating `"`?:
`{% tab title="iOS (ACP 2.x) %}` -> `{% tab title="iOS (ACP 2.x)" %}`